core/src/runtime/core_map.rs

```diff
@@ -222,40 +229,42 @@ impl<'a> CoreMapBuilder<'a> {
 
             // spawns a new background thread and initializes a core executor on
             // that thread.
-            let join = thread::spawn(move || {
-                debug!("spawned background thread {:?}.", thread::current().id());
+            let join = thread::Builder::new()
```

Comment:
this is prob fine, but double-checked formatter here?

Reply:
yep ran rustfmt.